modules/appeals_api/spec/fixtures/v2/valid_200995_headers.json

```diff
@@ -7,5 +7,8 @@
   "X-VA-File-Number": "987654321",
   "X-VA-Insurance-Policy-Number": "987654321123456789",
   "X-Consumer-Username": "va.gov",
-  "X-Consumer-ID": "some-guid"
+  "X-Consumer-ID": "some-guid",
```

Comment:
I _think_ we want these headers in the extra fixtures only. Maybe we need a _minimal_ 200995 fixture set and test??